.gitignore

```diff
@@ -1,6 +1,10 @@
 /target
 /local
 
+# hidden dir
+.*
```

Comment:
I think that ".\*" might have some unintended effects; probably best to make it mirror Optaplanner .gitignore: https://github.com/kiegroup/optaplanner/blob/master/.gitignore (which uses "/.*" instead (i.e. only ignore hidden files at root))